src/ripple/protocol/impl/STObject.cpp

```diff
@@ -116,7 +116,8 @@ bool STObject::setType (const SOTemplate& type)
                 JLOG (debugJournal().warn())
                     << "setType(" << getFName().getName()
                     << "): explicit default " << e->e_field.fieldName;
-                valid = false;
+                Throw<std::runtime_error>("Explicit default " +
+                    e->e_field.fieldName);
```

Comment:
What do you suggest then Vinnie? Something like returning a `std::pair<bool, std::string>`?